build.gradle

```diff
@@ -609,6 +609,9 @@ task product(type: Zip) {
     dependsOn ":snappy-aqp_${scalaBinaryVersion}:jar"
     targetProject = project(":snappy-aqp_${scalaBinaryVersion}")
   }
+  if (rootProject.hasProperty('enterprise')) {
+    dependsOn ":snappy-enterprise_${scalaBinaryVersion}:jar"
+  }
```

Comment:
Can change this property to "snappydata.enterprise" so that its cleaner to set in user's ~/.gradle/gradle.properties if required